### ISSUE TITLE
What does "basis" compound mean?

### ISSUE THREAD

**Issue by TranslatorIssueCreator** [NCATSTranslator/Feedback#567]
## Steps to reproduce:
MVP1 on Premature menaupose
UI CI
submitted by @sandrine-muller-research

**Comment by sandrine-muller-research** [comment]
The second top result is "basis" from RTX-KG2:
<img width="1261" alt="image" src="https://github.com/NCATSTranslator/Feedback/assets/22616936/6b7d255e-0c5d-4d6e-9bbd-544b14358bba">
The [reported source](http://linkedlifedata.com/resource/umls/id/C1874451) for the identifier is not clear to me so that I understand what it is. My feeling is that it is a Broad concept and should be in the Block list.

**Comment by gglusman** [comment]
Very strange indeed. That linkedlifedata page links to two papers with 'BASIS' in them..
1. In the first, BASIS appears to be an acronym for the [Bronchoscopy AntiSmoking Study](https://pubmed.ncbi.nlm.nih.gov/19726173/). The paper doesn't mention menopause, and the only link I can see is through bronchiectasis, which is mentioned as a comorbidity in the paper and is more frequent [post-menopause](https://www.atsjournals.org/doi/full/10.1513/AnnalsATS.202002-090ED).
2. In the second, BASIS stands for "Biology of Ageing e-Science Integration and Simulation"... a 2010 "system for model definition, simulation and visualization for stochastic models written in the SBML language".
To me this seems like acronym confusion in that source.

**Comment by dkoslicki** [comment]
Agreed: looks like acronym confusion and NLP messiness to me as well
